README.md

```diff
@@ -0,0 +1,5 @@
+* Explain how multi-dimentional arrys works?
+* Give an example of inheritance in js?
+* Do we have a trees and pointers in js?
+* Can we perform Multithreading in js?
+* how to define private and public classes and functions in js?

```

Comment:
Good question which requires you to demonstrate learning without being time-consuming to answer.